pom.xml

```diff
@@ -493,6 +498,36 @@
           <artifactId>jackson-databind</artifactId>
           <version>2.13.1</version>
         </dependency>
+        <dependency>
```

Comment:
For the mapr build, we are still using Hadoop 2.7.0-mapr-1602, all the new dependencies we added for Hadoop 3.3.0 are not needed here.

One further question: could you please check with Shiva whether we are still releasing mlcp MapR build? I think we have discontinued the support. It may make sense to refactor the pom files to get rid of MapR build when we have the bandwidth.

Thanks,
Vanessa

Reply:
Removed the newly added dependencies in mapr bindist